Makefile

```diff
@@ -6,11 +6,13 @@ PIP ?= pip
 VENV = virtualenv
 VENV_ARGS = -p $(PYTHON)
 VENV_DIR = $(CURDIR)/.venv
+VENV_DATA_DIR = $(CURDIR)/data
 
 all: install
 
 $(VENV_DIR): requirements-dev.txt
 	$(VENV) $(VENV_ARGS) "$(VENV_DIR)"
+	echo "export WATSON_DIR=\"$(VENV_DATA_DIR)\"" >> "$(VENV_DIR)"/bin/activate
```

Comment:
```suggestion
	echo "export WATSON_DIR=\"$(WATSON_DIR)\"" >> "$(VENV_DIR)"/bin/activate
```

Reply:
@jmaupetit what about `VENV_WATSON_DIR`? I find having two `WATSON_DIR` variables in two languages (shell & makefile) in the same expression a bit confusing. Besides, prefixing with `VENV` seems quite consistent and meaningful in that context. What do you think? If I didn't convince you I'll go with `WATSON_DIR` , no problem :)

Reply:
I find this more consistent, but `VENV_WATSON_DIR` is ok to me (at least more relevant than `VENV_DATA_DIR` :wink:).